perf/benchmark/security/generate_policies/generate_policies.go

```diff
@@ -185,14 +210,111 @@ func generatePeerAuthentication(policyData SecurityPolicy, policyHeader *MyPolic
 	return yaml, nil
 }
 
+func generateToken(policyData SecurityPolicy, privateKey *rsa.PrivateKey) (string, error) {
```

Comment:
probably it's time to refactor the layout of the code here as it grows larger, you can at least put the JWT related code in a separate file, and (optional) factor out the main() related code to a separate file.

Reply:
Refactored the jwt and jwks code to a seperate file